src/WebJobs.Script/ManagedDependencies/IManagedDependencyConfigs.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.Grpc.Messages;
+
+namespace Microsoft.Azure.WebJobs.Script.ManagedDependencies
+{
+    internal interface IManagedDependencyConfigs
```

Comment:
Is this an abstraction for the configuration used by the feature? This should use the `Options` pattern you'll find throughout the code instead.

Reply:
yes, by using this abstraction, I'm able to conditionally test my code and provide different test condition on local and in app service environment